ingest-app/docs/api.md

```diff
@@ -543,11 +543,14 @@ Granule metadata can be deleted by sending an HTTP DELETE the URL `%CMR-ENDPOINT
 A new variable ingest endpoint is provided to ensure that variable association is created at variable ingest time.
 Variable concept can be created or updated by sending an HTTP PUT with the metadata to the URL `%CMR-ENDPOINT%/collections/<collection-concept-id>/<collection-revision-id>/variables/<native-id>`.  `<collection-revision-id>` is optional. The response will include the [concept id](#concept-id),[revision id](#revision-id), variable-association and associated-item.
 
+Association data can be provided by using multiple `-d` flags with the data being sent as the content for the second `-d` section. When updating a Variable, the calling application is responible for updating any association data meaning that if the data is to be present in the new revision, it must be carried over from the previouse revision.
```

Comment:
typos. responible --> responsible. previouse --> previous

Reply:
Recommend to change `When updating a Variable, the calling application is responsible for updating any association data meaning that if the data is to be present in the new revision, it must be carried over from the previous revision.` to `The new association data will overwrite any existing association data in the previous revision`.